src/domain/safe/safe.repository.ts

```diff
@@ -34,382 +34,382 @@ export class SafeRepository implements ISafeRepository {
-      moduleTransactionId,
+      args.moduleTransactionId,
     );
     return this.moduleTransactionValidator.validate(moduleTransaction);
   }
 
-  async getModuleTransactions(
-    chainId: string,
-    safeAddress: string,
-    to?: string,
-    module?: string,
-    limit?: number,
-    offset?: number,
-  ): Promise<Page<ModuleTransaction>> {
+  async getModuleTransactions(args: {
+    chainId: string;
+    safeAddress: string;
+    to?: string;
+    module?: string;
+    limit?: number;
+    offset?: number;
+  }): Promise<Page<ModuleTransaction>> {
     const transactionService =
-      await this.transactionApiManager.getTransactionApi(chainId);
-    const page = await transactionService.getModuleTransactions(
-      safeAddress,
-      to,
-      module,
-      limit,
-      offset,
-    );
+      await this.transactionApiManager.getTransactionApi(args.chainId);
+    const page = await transactionService.getModuleTransactions({
+      safeAddress: args.safeAddress,
+      to: args.to,
+      module: args.module,
+      limit: args.limit,
+      offset: args.offset,
+    });
     return this.moduleTransactionValidator.validatePage(page);
   }
 
-  async clearModuleTransactions(
-    chainId: string,
-    safeAddress: string,
-  ): Promise<void> {
+  async clearModuleTransactions(args: {
+    chainId: string;
+    safeAddress: string;
+  }): Promise<void> {
     const transactionService =
-      await this.transactionApiManager.getTransactionApi(chainId);
+      await this.transactionApiManager.getTransactionApi(args.chainId);
 
-    return transactionService.clearModuleTransactions(safeAddress);
+    return transactionService.clearModuleTransactions(args.safeAddress);
   }
 
-  async getTransactionQueue(
-    chainId: string,
-    safe: Safe,
-    limit?: number,
-    offset?: number,
-  ): Promise<Page<MultisigTransaction>> {
-    return this._getTransactionQueue(
-      chainId,
-      safe,
-      'nonce,submissionDate',
-      limit,
-      offset,
-    );
+  async getTransactionQueue(args: {
+    chainId: string;
+    safe: Safe;
+    limit?: number;
+    offset?: number;
+  }): Promise<Page<MultisigTransaction>> {
+    return this._getTransactionQueue({
+      chainId: args.chainId,
+      safe: args.safe,
+      ordering: 'nonce,submissionDate',
+      limit: args.limit,
+      offset: args.offset,
+    });
   }
 
-  async getTransactionQueueByModified(
-    chainId: string,
-    safe: Safe,
-    limit?: number,
-    offset?: number,
-  ): Promise<Page<MultisigTransaction>> {
-    return this._getTransactionQueue(chainId, safe, '-modified', limit, offset);
+  async getTransactionQueueByModified(args: {
+    chainId: string;
+    safe: Safe;
+    limit?: number;
+    offset?: number;
+  }): Promise<Page<MultisigTransaction>> {
+    return this._getTransactionQueue({
+      chainId: args.chainId,
+      safe: args.safe,
+      ordering: '-modified',
+      limit: args.limit,
+      offset: args.offset,
+    });
```

Comment:
What do you think about spreading `args` in these?

Reply:
[9db8374](https://github.com/safe-global/safe-client-gateway-nest/pull/610/commits/9db8374f600be9d8fbe58d5c91761aef0ea8b5cf)